### PR TITLE
Rename Azure Active Directory to Microsoft Entra ID

### PR DESCRIPTION
**Pull request by GGP1**
## Description

Updated the Azure Active Directory terminology according to Microsoft's new [glossary](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/new-name#glossary-of-updated-terminology). 

Following their [decision](https://devblogs.microsoft.com/identity/aad-rebrand/), I haven't updated directory names, URLs or file names to avoid breaking links to the documentation.

Furthermore, the Azure Active Directory Graph API is deprecated but it is still operational and shouldn't be changed.

### Docs building
- [x] Compiles without warnings.

<details><summary>build</summary>

```console
Running Sphinx v7.0.1
loading translations [en-US]... not available for built-in messages
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 1 source files that are out of date
updating environment: 0 added, 1 changed, 0 removed
reading sources... [100%] cloud-security/azure/activity-services/active-directory/index                                                                                             
Copying static files for wazuh-doc-images...[100%] img/loading.gif                                                                                                                  
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index                                                                                                                                                      
generating indices... done
writing additional pages... not_found moved-content search done
copying images... [100%] images/cloud-security/azure/aad-graph-intro.png                                                                                                            
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in home/python/build/html.
[sphinx-autobuild] Serving on http://0.0.0.0:8000
```

</details>

### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/user-manual/reference/ossec-conf/wodle-azure-logs.rst [hunk 1]
```diff
@@ -475,7 +475,7 @@ Identifier of the application that we will use for the authentication and to be
 graph\\application_key
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Key to the application we will use for authentication and to be able to use the AAD Graph API. It must be used next to the ``application_id`` option obligatorily. Incompatible with ``auth_path`` option.
+Key to the application we will use for authentication and to be able to use the Azure Active Directory Graph API. It must be used next to the ``application_id`` option obligatorily. Incompatible with ``auth_path`` option.
```
Comment (GGP1) [inline source/user-manual/reference/ossec-conf/wodle-azure-logs.rst:478]: As I mentioned in the PR description, the Azure Active Directory Graph API is still operational and it is different from the Microsoft Graph API

**Review by Selutario — CHANGES_REQUESTED** [review]
LGTM. However, since Microsoft already renamed their service, the sooner we update our documentation the better. Therefore, we should merge this change in [4.5](https://github.com/wazuh/wazuh-documentation/tree/4.5), which is the production branch.

**Comment by Selutario** [comment]
Sorry @GGP1, we published 4.6.0 and this was not approved & merged in 4.5, so it needs to be rebased (again) to 4.6. Let me know as soon as it is ready again so I approve it.

**Review by Selutario — APPROVED** [review]

File: source/user-manual/user-administration/index.rst [hunk 2]
```diff
@@ -12,7 +12,7 @@ In the password management section, you can find instructions on how to use the
 
 The RBAC section contains directions on how to create Wazuh indexer users, also known as internal users, assign them different roles and map them to the Wazuh manager API. Find out how to create an admin user, a read-only user, a custom user, and a user with permission to read and manage only a group of agents. 
 
-In the single sign-on section, you can find instructions on how to integrate Wazuh with different Identity Providers to implement Single Sign-On. Find instructions for Okta, Azure Active Directory, PingOne, Google, Jumpcloud and OneLogin. 
+In the single sign-on section, you can find instructions on how to integrate Wazuh with different Identity Providers to implement Single Sign-On. Find instructions for Okta, Microsoft Entra ID, PingOne, Google, Jumpcloud and OneLogin. 
 
 In the LDAP integration section, you can find instructions on how to integrate Wazuh with LDAP/Active Directory to authenticate and authorize users.
```
Comment (javimed) [inline source/user-manual/user-administration/index.rst:17]: Please consider the following "Active Directory" occurrences, if applicable:

- Up here, where it says "_LDAP/**Active Directory** to authenticate and authorize users_".
- In RBAC `/user-manual/user-administration/rbac` section.
- In the new LDAP `/user-manual/user-administration/ldap` section.
-  In `/user-manual/user-administration/single-sign-on/` sections. Please consider a url change and redirection for the `azure-active-directory` page.

**Review by javimed — COMMENTED** [review]
Please also consider changing this if applicable

image

**Review by javimed — APPROVED** [review]

**Pull request merged**